### PR TITLE
docs(context): Add server-level context and API links

### PR DESCRIPTION
**Pull request by kjdelisle**
### Description
- Explain server-level context with examples
- Add link to API Docs to find more info on using the Binding functions.

### Related Issues
connected to https://github.com/strongloop/loopback-next/issues/651

### CODE REVIEW

File: pages/en/lb4/Context.md [hunk 1]
```diff
@@ -48,6 +51,42 @@ In this case, you are using the `.controller` helper method to register a new
 controller. The important point to note is `MyController` is actually registered
 into the Application Context (`app` is a Context).
 
+## Server-level context (components that make use of Server)
+- has the Application-level context as a parent
+- holds configuration specific to this particular server instance
```
Comment (crandmck) [inline pages/en/lb4/Context.md]: Server-level context:
- Is a child of application-level context.
- Contains configuration for a particular server instance.

---
Also, what does the parent/child relationship mean here?  I think some more explanation is in order.

**Review by crandmck — COMMENTED** [review]
Please be consistent with capitalization of "Application," "Server," and "Request".  My preference would be lowercase; or if it refers to actual code objects, use uppercase and make `monospace` with back-ticks to indicate that.

File: pages/en/lb4/Context.md [hunk 2]
```diff
@@ -48,6 +51,35 @@ In this case, you are using the `.controller` helper method to register a new
 controller. The important point to note is `MyController` is actually registered
 into the Application Context (`app` is a Context).
 
+## Server-level context (components that make use of Server)
+- Is a child of application-level context
```
Comment (jannyHou) [inline pages/en/lb4/Context.md]: I think you need an extra line here to separate it from the second line?

**Review by jannyHou — APPROVED** [review]

File: pages/en/lb4/Context.md [hunk 3]
```diff
@@ -48,6 +51,36 @@ In this case, you are using the `.controller` helper method to register a new
 controller. The important point to note is `MyController` is actually registered
 into the Application Context (`app` is a Context).
 
+## Server-level context
+
```
Comment (crandmck) [inline pages/en/lb4/Context.md:55]: Also add "Server-level context:" as a lead-in paragraph... It makes complete sentences that are completed by the two bullet-points (the heading is separate).

**Review by crandmck — DISMISSED** [review]
Otherwise, LGTM

**Pull request merged**